compiler/src/iree/compiler/Preprocessing/Common/ConvertConvFilterToChannelsLast.cpp

```diff
@@ -104,6 +104,122 @@ struct ConvertHwcfToFhwc : public OpRewritePattern<linalg::Conv2DNhwcHwcfOp> {
   }
 };
 
+struct ConvertChwfToFhwc : public OpRewritePattern<linalg::GenericOp> {
```

Comment:
It looks like a generic implementation. Why do we name it as `ChwfToFhwc`?

If we have generic implementation, do we still need `ConvertHwcfToFhwc` pattern?